en/guides/manage-devices/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Introduction
+table_of_contents: true
+---
+
+# Introduction
+
+On some systems it is appropriate to have a system user. For example, with a system user, one can ssh to the device and run commands. (A system user is not required.)
+
+By default on first boot, stock core images finally run a program called console-conf. If the user has a keyboard and monitor attached to the device, they can use console-conf to configure the network and add a user account. This user account can then be used to ssh to the device for various management purposes.
+
+Some systems suppress console-conf, and a user is not created by default. In these cases, one may add a system-user if one has the information and authority needed to do so for the particular system. This is done by inserting a USB key with a special file (named `auto-import.assert`) in the root directory. Snapd imports the assertions found in it, and, if the assertions are valid for the given system, the system-user is created. After this, you can log into the device (locally or over ssh) using the username and password defined.
+
+In general, you cannot add a system-user to a system that is already "managed". A system is managed if a user was already created by console-conf, or if a system-user was created through the auto-import mechanism described here.
+
+Whether you can add a system-user to an unmanaged system depends on the model assertion that was used to create the image. It also depends on whether you have an authorized snapcraft key on your local system.
```

Comment:
The current discussion of keys that I found is in a section about Ubuntu Images. https://docs.ubuntu.com/core/en/guides/build-device/image-building

I've added the link now, BUT:

I think that is the wrong place for a introduction and discussion of snapcraft keys. Keys should be a free standing topic not contained within image building. Adding the link raises the possibility that it will break later when/if the new topic is created.

I don't think the documentation-builder tool finds broken links. This should be fixed for obvious doc quality and maintenance reasons,